tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -370,7 +370,9 @@ After Docker is installed, launch a Docker container with the TensorFlow binary
 image as follows.
 
 ```bash
-$ docker run -it -p 8888:8888 gcr.io/tensorflow/tensorflow
+$ docker run -d -p 8888:8888 --name="tensor" gcr.io/tensorflow/tensorflow
+
+$ docker exec -it tensor bash
```

Comment:
Maybe keep both sets of instructions:

"To start the Notebook App: 
$ docker run -it -p 8888:8888 gcr.io/tensorflow/tensorflow

To start a command-line interface:
$ docker run -it -p 8888:8888 --entrypoint /bin/bash gcr.io/tensorflow/tensorflow
"

(I think --entrypoint /bin/bash might be nicer in this case -- does that work for you?).